build/zip_bundle.gni

```diff
@@ -19,11 +19,25 @@ if (flutter_runtime_mode == "jit_release") {
 # The files variable is an array of scopes that specify a source file or
 # directory and a destination path in the archive to create.
 #
+# If this zip file is not intended to be codesigned as a Mac artifact,
+# set variable encode_entitlements to false.
+#
+# If encode_entitlements is set to true, this api will embed Mac
+# codesign entitlements information in the zip file at
+# $root_build_dir/zip_archives.
+# If encode_entitlements is true, user provides two array variables
+# 'entitlements' and 'without_entitlements'.
+# The entitlements array specifies the list of filepaths that should be
+# codesigned with entitlements.
+# The without_entitlements array specifies the list of filepaths that
```

Comment:
This seems a bit contradictory. Is this still signing but signing without entitlements?